win/WIN_ToggleButton.h

```diff
@@ -25,7 +25,7 @@ namespace win
 
 		void draw() override;
```

Comment:
Another general comment which may be irrelevant due to activity on otther branches. If we've abstracted rendering, there is no need to pass renderers in constructors. The draw methods should be protoyped

`virtual void draw(Renderer * renderer)`

Where the `Renderer` type is the abstracted one. Then the `renderer_` field can be removed from the `UIelement` subclasses.
**COULD**

Reply:
will revisit once I'm up to date with other branches